src/ADAL.PCL/AuthenticationResult.cs

```diff
@@ -74,6 +74,19 @@ internal AuthenticationResult(string accessTokenType, string accessToken, DateTi
         public DateTimeOffset ExpiresOn { get; internal set; }
 
         /// <summary>
+        /// Gets the point in time in which the Access Token returned in the AccessToken property ceases to be valid in ADAL's extended LifeTime.
+        /// This value is calculated based on current UTC time measured locally and the value ext_expiresIn received from the service.
+        /// </summary>
+        [DataMember]
+        public DateTimeOffset ExtendedExpiresOn { get; internal set; }
```

Comment:
why is this public? why should a developer care when there is a Boolean already that denotes it?